Import documentation with screenshots for HITL (#54618)
mistercrunch/Airflow: airflow-core/docs/tutorial/hitl.rst

   :start-after: [START howto_hitl_operator]
   :end-before: [END howto_hitl_operator]

.. image:: /img/hitl_wait_for_option.png

>   :alt: Demo HITL task instance waiting for option

Reviewer: ```suggestion
  :alt: Demo HITL task instance waiting for an option
```

Author: Sure! Thanks for pointing out.

   :start-after: [START howto_hitl_workflow]
   :end-before: [END howto_hitl_workflow]

.. image:: /img/hitl_branch_selection.png
  :alt: Demo HITL task instance waiting for branch selection

After the branch is chosen, the workflow will proceed along the selected path.

.. image:: /img/hitl_branch_selected.png
  :alt: Demo HITL task instance after branch selection

Notifiers
---------

A notifier is a callback mechanism that allows you to handle HITL events, such as when a task is waiting for human input, succeeds, or fails.
The example uses a notifier ``LocalLogNotifier`` that logs messages for demonstration.

> You can implement your own notifier for different functionalities.

Reviewer 2: https://airflow.apache.org/docs/apache-airflow/stable/howto/notifications.html

https://airflow.apache.org/docs/apache-airflow-providers/core-extensions/notifications.html

We probably could mentione these 2 links here

Author: Sure, adding links would be definitely helpful.

> In the example DAG, the notifier is defined as follows:

Reviewer 2: ```suggestion
In the example Dag, the notifier is defined as follows:
```

Author: Sorry about the miss. Have fixed it.

.. exampleinclude:: /../../providers/standard/src/airflow/providers/standard/example_dags/example_hitl_operator.py
   :language: python
   :start-after: [START hitl_notifier]
   :end-before: [END hitl_notifier]

You can pass a list of notifiers to HITL operators using the ``notifiers`` argument as follows.

> When the operator reaches certain states, the ``notify`` method will be called.

Reviewer 2: ```suggestion
When the operator creates an HITL request that is waiting for a human response, the `notify` method will be called with a single argument, `context`.
```

Author: I use double backticks for `notify` and `context ` since double backtick is needed for code in RST files.
